pkg/gpg/gpg_forwarding.go

```diff
@@ -88,6 +89,19 @@ func (g *GPGConf) ImportOwnerTrust() error {
 	return gpgOwnerTrustCmd.Run()
 }
 
+func (g *GPGConf) SetupGpgGitKey() error {
+	if g.GitKey != "" {
+		gitConfigCmd := exec.Command("git", []string{"config", "--global", "user.signingKey", g.GitKey}...)
```

Comment:
another small nit: can this go into the `gitcredentials` package? For the sole reason of keeping one place where we set the git config

Reply:
Done :+1: